docs/en/stack/security/authorization/privileges.asciidoc

```diff
@@ -48,6 +48,9 @@ roles of the user who created or updated them.
 `manage_pipeline`::
 All operations on ingest pipelines.
 
+`manage_rollup`::
+All rollup operations.
```

Comment:
", such as creating, starting, stopping and deleting rollup jobs, as well as all read-only operations."